obonet/io.py

```diff
@@ -24,7 +24,10 @@ def open_read_file(path):
             content = response.read()
         if opener == io.open:
             encoding = response.headers.get_content_charset()
```

Comment:
We [should be able](https://stackoverflow.com/a/24372670/4651668) to set the default encoding of `utf-8` in case of a missing header in `response.headers.get_content_charset()`.

Since this is a bugfix for https://github.com/dhimmel/obonet/pull/7#commitcomment-25146737 that's not related to networkx v2.0 compatibility, it would be best to make this change as a distinct PR. This will help keep the commit history clean.